aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1041,6 +1040,8 @@ module aptos_token::token {
         assert!(string::length(&name) <= MAX_NFT_NAME_LENGTH, error::invalid_argument(ENFT_NAME_TOO_LONG));
         assert!(string::length(&collection) <= MAX_COLLECTION_NAME_LENGTH, error::invalid_argument(ECOLLECTION_NAME_TOO_LONG));
         assert!(string::length(&uri) <= MAX_URI_LENGTH, error::invalid_argument(EURI_TOO_LONG));
+        assert!(royalty_points_numerator <= royalty_points_denominator, error::invalid_argument(EINVALID_ROYALTY_NUMERATOR_DENOMINATOR));
```

Comment:
do we also want to check that the denominator is non-zero?

Reply:
no, currently, the 0 should already be handled by eco projects. If we enforce the denominator to be non zero, it will create unnecessary txn failures

Reply:
ok makes sense

Reply:
Is 100% (numerator == denominator) even a valid value? To put it a different way, is there a right value for what the max royalty should be?

Reply:
The purpose is to prevent malicious behavior where buyer can lose more money than the price. Whether 100% or 90% is valid, I think it shouldn't be enforced in the standard

Reply:
I think we should at least validate that numerator is strictly less than denominator. 100% royalty seems like an error to me and should not be possible.

Reply:
1. how "0 ise handled by eco projects"?
2. I think 100% is fine.

Reply:
How is 100% fine? That means that the transfer's amount will go 100% to the creator?

Reply:
can we actually enforce this at this point in time?
someone might have a contract out here that assumes this assert doesn't exist.
If we want to make this, we need to submit an AIP to the community and give at least a weeks worth of feedback. It is a breaking change.